library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -175,7 +178,16 @@ libspdm_return_t libspdm_receive_response(void *context, const uint32_t *session
     /* always use scratch buffer to response.
      * if it is secured message, this scratch buffer will be used.
      * if it is normal message, the response ptr will point to receiver buffer. */
-    libspdm_get_scratch_buffer (spdm_context, response, response_size);
+    transport_header_size = spdm_context->transport_get_header_size(spdm_context);
+    libspdm_get_scratch_buffer (spdm_context, (void **)&scratch_buffer, &scratch_buffer_size);
+    *response = scratch_buffer + transport_header_size;
+    #if LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP
+    *response_size = scratch_buffer_size - transport_header_size - LIBSPDM_MAX_SPDM_MSG_SIZE -
+                     LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE;
+    #else
+    *response_size = scratch_buffer_size - transport_header_size
+    #endif
+
```

Comment:
Please add comment into the code:
The rang of 0 ~ LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE is for Secure encoding/
decoding, LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE ~ End is for Chunk.